packages/TermsAndConditions/FootnoteLink/FootnoteLink.md

```diff
@@ -1,9 +1,9 @@
 ### Usage criteria
 
-- If there is only one annotation needed on a page, try including it within the copy on the page and avoid using `FootnoteLink`
-- For [PriceLockup](#pricelockup) all the numbers should appear together below the `bottomText`
-- Must be wrapped by a `Text`, `Heading`, or `DisplayHeading` component
-  - When using `FootnoteLink` in a `PriceLockup`, `PriceLockup` will automatically wrap all `bottomText` in a `Text` component
+- `FootnoteLink` inherits parent element’s text size and colour
+- Avoid using `FootnoteLink` if there is only one annotation on a page, include it as part of the content
```

Comment:
Adding an "and" after the comma here might make this line read better.

"Avoid using `FootnoteLink` if there is only one annotation on a page, and include it as part of the content"